### PR TITLE
Added FAQ

### PR DESCRIPTION
**Pull request by markusborer**

### CODE REVIEW

File: doc/faq.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# FAQ
+
+## What is PostregSQL needed for?
+PostgreSQL is needed to safe temporary data about the running transaction.
```
Comment (Ghabi-ISC) [inline doc/faq.md]: to save

File: doc/faq.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## What is RabbitMQ needed for?
+RabbitMQ is a message queue. All received data (over REST) will be sent to RabbitMQ.
+A secondary process will then consume these messages and create the files for sedex.
+
+## What is Prometheus/Grafana needed for?
+Prometheus collects information from the running process and prepares them for Grafana.
+Grafana shows the collected information in a dashboard.
+
+They do not necessarily have to be installed, but they are helpful to get some insights in the running system.
```
Comment (Ghabi-ISC) [inline doc/faq.md]: These components are optional.

File: doc/faq.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+## Do I need to backup PostregSQL?
+No, because it is only needed for temporary data.
+
+## What do I have to do if the processor crashes during a transaction?
+If the processor crashes during a transaction it is best to reset the transaction and start a new one with the same person-data.
```
Comment (Ghabi-ISC) [inline doc/faq.md]: Does this mean that the transactions are idempotent. Repeating them wouldn't influence the index on LGS side?
Could you also clarify what "reset" means? Is it just starting a new transaction as if the previous one was not executed? or is there something to do/configure/call before running the new transaction?

**Review by Ghabi-ISC — CHANGES_REQUESTED** [review]
I've added few questions about the meaning where it could be confusing for some customers.

File: doc/faq.md (hunk 1 above)
Comment (markusborer) [reply doc/faq.md]: Corrected

File: doc/faq.md (hunk 2 above)
Comment (markusborer) [reply doc/faq.md]: Added

File: doc/faq.md (hunk 3 above)
Comment (markusborer) [reply doc/faq.md]: Refined the text

**Review by Ghabi-ISC — APPROVED** [review]

**Pull request merged**